### PR TITLE
Integration tests for extractRadiomics, generateScan

### PR DESCRIPTION
**Pull request by aauker**
1. Factors out some convenience methods (which need more work)
2. Tests generation of mhd and nrrd via params
3. Tests extraction of radiomics with two different bin widths
4. All tests compare checksums of outputs -> cannot do, as hashes are different
5. Test data from https://wiki.cancerimagingarchive.net/display/Public/QIN-Breast

### CODE REVIEW

**Comment by aauker** [comment]
Eventually able to catch this through a rather cryptic process:

exception = KeyError('GRAPH_URI',), exit_code = 1, prog_name = 'cli'

File: data_processing/scanManager/extractRadiomics.py [hunk 1]
```diff
@@ -50,43 +37,70 @@ def cli(cohort_id, container_id, method_id):
...
+    else:
+        return input_nodes[0].data()
+
+def get_method_data(method_id):
+    with open(f'{method_id}.json') as json_file:
+        method_config = json.load(json_file)['params']
+    return method_config
+
+def add_container_data(container_id, n_meta):
+    conn = Neo4jConnection(uri=os.environ["GRAPH_URI"], user="neo4j", pwd="password")
```
Comment (doori) [inline data_processing/scanManager/extractRadiomics.py:50]: Perhaps we can move connection closing outside of query(), and have us manually close() when we are done with the graph connection. This way, we don't need to establish the connection multiple times in the same process.

**Review by doori — APPROVED** [review]
Very nice! The refactoring in this PR makes the code much more readable and clean. Great addition of tests and test data.

File: data_processing/scanManager/extractRadiomics.py (hunk 1 above)
Comment (aauker) [reply data_processing/scanManager/extractRadiomics.py:50]: Agreed, managing the DB connection correctly would be a good next step!

**Pull request merged**